Library/Formula/zero-install.rb

```diff
@@ -0,0 +1,132 @@
+class ZeroInstall < Formula
+  desc "Zero Install is a decentralised software installation system"
+  homepage "http://0install.net/"
+  url "https://downloads.sf.net/project/zero-install/0install/2.8/0install-2.8.tar.bz2"
+  sha256 "12de771be748bce9350c90bc4720029a566b078ceabd335af09386ac6a37df2b"
+
+  depends_on :gpg
+  depends_on "curl"
+  depends_on "gtk+" => :optional
+  depends_on "pkg-config" => :build
+  depends_on "ocaml" => :build
+  depends_on "opam" => :build
```

Comment:
It's not really that important, but for future referencing sake, we prefer something like:

```ruby
depends_on "" => :build
depends_on "" => :build
depends_on ""
depends_on "" => :optional
depends_on :special_dep
```

Build deps, then mandatory deps, then optional deps, then special deps.